doc/source/ray-air/doc_code/air_key_concepts.py

```diff
@@ -35,6 +35,8 @@
     scaling_config=ScalingConfig(
         num_workers=num_workers,
         use_gpu=use_gpu,
+        # Make sure to leave some CPUs free for Ray Data operations.
+        _max_cpu_fraction_per_node=0.9,
```

Comment:
do we really need this in the starter code @Yard1?

Reply:
@amogkam it gets stuck otherwise

Reply:
If it's common then let's just make 0.9 the default? If it's not common and this is just to get CI to pass, then let's increase the # of CPUs in the CI machine.

The quick start code should be quick start...ideally not requiring additional private configurations to get things to work.

Reply:
But will leave this up to you

Reply:
I think setting it to default would be a good idea, but that should be definitely done in a separate PR as it will have wide impact.